Release v1.6.0

m3r1v3 opened the pull request:
## Release v1.6.0
In this release need to update project icon, change colors, add Material You icon, replace encryption algorithm, etc.

## Task list 
- [x] #57 
- [x] #58 
- [x] #59 
- [x] #60
- [x] #61

m3r1v3 committed:
Update Securely icon

:sparkles: feat: Update Securely icon

m3r1v3 committed:
Update Gradle version

:hammer: build: Update Gradle version

m3r1v3 committed:
Update Securely logo

:sparkles: feat: Update Securely logo

m3r1v3 committed:
Update build.gradle

:hammer: build: Update build.gradle

m3r1v3 committed:
Update app launch icon

:sparkles: feat: Update app launch icon
:sparkles: feat: Add support of Material You app icon

m3r1v3 committed:
Update AndroidManifest.xml

:hammer: build: Update AndroidManifest.xml

m3r1v3 committed:
Update project colors

:sparkles: feat: Update project colors

m3r1v3 committed:
Fix build gradle error due to Room dependency

:bug: fix: Fix build gradle error due to Room dependency
:hammer: build: Update Gradle version to 7.4.1

m3r1v3 committed:
Update Securely icon

:sparkles: feat: Update Securely icon

m3r1v3 committed:
Refactor AndroidManifest.xml

:recycle: refactor: Refactor AndroidManifest.xml

m3r1v3 committed:
Update Securely icon (Finally)

:sparkles: feat: Update Securely icon (Finally)

m3r1v3 committed:
Update version code in build.gradle

:hammer: build: Update version code in build.gradle

m3r1v3 committed:
Update project icon (Final 😂)

:sparkles: feat: Update project icon (Final 😂)

merive commented:
It's happened! We made nice icon and now start to work with other issues. It was hard, but we did it 😆

merive commented:
> It's happened! We made nice icon and now start to work with other issues. It was hard, but we did it 😆

Now we will work on issue #61. Need to connect application to API (released ~some months ago~ recently). It will ~make workable~ improve UpdateFragment (add changelog, male getting version code faster)

m3r1v3 committed:
Update Securely icon design (finally fckin 🤬)

:sparkles: feat: Update Securely icon design

m3r1v3 committed:
Remake fckin Securely icon

:sparkles: feat: Remake fckin Securely icon

m3r1v3 committed:
Create API class in api package

:sparkles: feat: Create API class in api package

m3r1v3 committed:
Update Securely icon (for fckin thousandth time)

:sparkles: feat: Update Securely icon

m3r1v3 committed:
Remove transition tag in activity_main_scene.xml

:recycle: refactor: Remove transition tag in activity_main_scene.xml

m3r1v3 committed:
Return transition tag in activity_main_scene.xml

:recycle: refactor: Return transition tag in activity_main_scene.xml

m3r1v3 committed:
Connect application to API

:sparkles: feat: Connect application to API
:sparkles: feat: Update fragment_update layout
:sparkles: feat: Rewrite code in Update Fragment
:sparkles: feat: Update checkVersion() method in MainActivity
:recycle: refactor: Remove unnecessary methods in MainActivity

m3r1v3 committed:
Lock activity orientation in AndroidManifest.xml

:recycle: refactor: Lock activity orientation in AndroidManifest.xml

m3r1v3 committed:
Move Password Recycler View to PasswordListFragment

:recycle: refactor: Move Password Recycler View to PasswordListFragment
:sparkles: feat: Create PasswordListFragment class and fragment_password_list layout
:recycle: refactor: Remove password_recycler_view from activity_main layout

m3r1v3 committed:
Update fragment_empty layout

:sparkles: feat: Update fragment_empty layout

m3r1v3 committed:
Change accent color (green to pale green)

:sparkles: feat: Change accent color (green to pale green)
:recycle: refactor: Remove @RequiresApi annotations

m3r1v3 committed:
Remove marginTop for line in activity_main layout

:recycle: refactor: Remove marginTop for line in activity_main

m3r1v3 committed:
Change some fragment UI design

:sparkles: feat: Change some fragment UI design
:sparkles: feat: Change radius value

m3r1v3 committed:
Change UI in some fragment layouts

:sparkles: feat: Change UI in some fragment layouts

m3r1v3 committed:
Update string.xml values

:sparkles: feat: Update string.xml values

m3r1v3 committed:
Update activity_login and fragment_password layouts

:sparkles: feat: Update activity_login and fragment_password layouts

m3r1v3 committed:
Update Encryption/Decryption algorithms

:sparkles: feat: Update Encryption/Decryption algorithms
:recycle: refactor: Rename key to password in application context

m3r1v3 committed:
Disable allowBackup in AndroidManifest.xml

:sparkles: feat: Disable allowBackup in AndroidManifest.xml

m3r1v3 committed:
Fix exported error in AndroidManifest.xml

:bug: fix: Fix exported error in AndroidManifest.xml
